config/config.go

```diff
@@ -255,9 +266,9 @@ func NewConfig() (c *Config) {
 		log.Error("Grafana username (\"GRAFANA_USERNAME\") requires that Grafana password (\"GRAFANA_PASSWORD\") is set.")
 	}
 
-	// Jaeger Configuration
-	c.ExternalServices.Jaeger.URL = strings.TrimSpace(getDefaultString(EnvJaegerURL, ""))
-	c.ExternalServices.Jaeger.Service = strings.TrimSpace(getDefaultString(EnvJaegerService, "jaeger-query"))
+	// Tracing Configuration
+	c.ExternalServices.Tracing.EnableJaeger = false
```

Comment:
this is hardcoded to false - it isn't coming from any config setting. a comment here explaining why this is set differently than all the rest (e.g. with getDefaultBoolean) would be helpful.

Reply:
This must be false until discover